frontendActions/src/main/resources/actions/messages/GitMacheteBundle.properties

```diff
@@ -0,0 +1,97 @@
+action.description.disabled.branch.status={0} disabled because the branch is {1}
+action.description.disabled.undefined.branch.name={0} disabled due to undefined branch name
+action.description.disabled.undefined.git-machete-repository=Action disabled due to undefined Git Machete repository
+action.description.disabled.undefined.machete.branch={0} disabled due to undefined machete branch
+action.description.disabled.undefined.sync-to-remote={0} disabled due to undefined sync to remote status
+
+# used indirectly within ISyncToRemoteStatusDependentAction
+action.description.enabled={0} branch ''{1}''
+
+action.text.current-branch={0} Current Branch
+
+action.checkout.description.disabled.currently-checked-out=Branch ''{0}'' is currently checked out
+action.checkout.description=Checkout branch ''{0}''
+action.checkout.task.title=Checking out
+
+action.fast-forward.description=Fast forward branch ''{0}'' to match ''{1}''
+action.fast-forward.description.root.branch=Root branch ''{0}'' cannot be fast-forwarded
+action.fast-forward.task.title=Fast Forwarding...
+action.fast-forward.text.current-branch=_Fast Forward Parent To Match Current Branch
+
+action.fetch.description=Fetch all remotes
```

Comment:
This string (and probably a few others) are repeated b/w here and ResourceBundle.properties